api/src/pcapi/core/users/api.py

```diff
@@ -1368,3 +1373,88 @@ def notify_users_before_deletion_of_suspended_account() -> None:
                 "Could not send email before deletion of suspended account",
                 extra={"user": account.id},
             )
+
+
+def anonymise_user(user: users_models.User, *, force: bool = False) -> None:
+    """
+    Anonymise the givent User. If force is True, the function will anonymise the user even if it has an adress and we
+    cannot find an iris for it.
+    """
+    iris = None
+    if user.address:
+        try:
+            iris = get_iris_from_address(address=user.address, postcode=user.postalCode)
+        except geography_exceptions.AddressApiError as exc:
+            logger.error("Could not anonymise user %s: %s", user.id, exc)
+            return
+        except geography_exceptions.AddressNotFound:
+            if not force:
+                return
+
+    # clean DCP on partner's side
```

Comment:
DCP ?

Reply:
De la même page en anglais : `Personal data, also known as personal information or personally identifiable information (PII), is any information related to an identifiable person.`
Mais je pense pas que PII soit mieux...

`personal data` ne suffit pas ?

Reply:
aller